### PR TITLE
Add usage examples to orb.yml

### PR DESCRIPTION
**Pull request by prashcr**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

Fixes https://github.com/CircleCI-Public/browser-tools-orb/issues/5

### Description

Adds usage examples to orb.yml, so they show up in the orbs registry.

### CODE REVIEW

**Comment by prashcr** [comment]
I wasn't sure if you noticed the Firefox/Geckodriver example. Let me know if I should keep it.

File: src/examples/install-latest-chrome.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  Example job that installs the latest version of Google Chrome
+
+usage:
+  version: 2.1
+  orbs:
+    browser-tools: circleci/browser-tools@0.1.3
+  workflows:
+    main:
+      jobs:
+        - browser-tools/install-chrome
```
Comment (KyleTryon) [inline src/examples/install-latest-chrome.yml]: There is no install-chrome job in this orb currently. There are only commands which could be utilized within a job.

https://circleci.com/docs/2.0/jobs-steps/

File: src/examples/install-specific-firefox-and-geckodriver.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  Example job that installs specific versions of Firefox and Geckodriver
+
+usage:
+  version: 2.1
+  orbs:
+      browser-tools: circleci/browser-tools@0.1.3
+  workflows:
+    main:
+      jobs:
+        - browser-tools/install-browser-tools:
```
Comment (KyleTryon) [inline src/examples/install-specific-firefox-and-geckodriver.yml]: Again here, this is referencing a non-existant job. The job would need to be defined within this config and then utilize the commands from this orb.

Here is an example of another orb which only contains commands. The usage example shows the creations of a job which utilizes these commands. https://circleci.com/orbs/registry/orb/circleci/aws-cli#usage-install_aws_cli

**Comment by KyleTryon** [comment]
Including in next major update. Thank you. Releasing soon.

**Pull request closed without merging**